test/dataset/test_feature_transforms.py

```diff
@@ -0,0 +1,20 @@
+import tempfile
+import torch
+
+from lhotse import CutSet
+from lhotse.dataset.feature_transforms import GlobalMVN
+
+
+def test_global_mvn_initialization_and_stats_saving():
```

Comment:
it'd be good to assert the shapes too just to make sure we're using the right object types (helps safe-guard against accidental comma's in returns etc., happened to me more than once in Python)